docs/src/components/landing/ToolpadHeroDemo.js

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+// import Box from '@mui/material/Box';
+import ToolpadHeroContainer from '../../layouts/ToolpadHeroContainer';
+
+export default function ToolpadHero() {
+  return (
+    <ToolpadHeroContainer
+      sx={{
+        backgroundRepeat: 'no-repeat',
+        backgroundPositionX: 'left',
+        backgroundSize: 'contain',
+        backgroundPositionY: {
+          xs: 'center',
+          sm: 'center',
+          md: 'top',
+          lg: 'top',
+        },
+        backgroundImage: 'url("/static/branding/product-toolpad-app.svg")',
```

Comment:
Also, the SVG format, I don't think that it makes sense since it has this inside:

```html
<image id="image0_997_56521" width="2839" height="1620" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAACxcAAAZUCAYAAAApFx4cAAAMP2lDQ1BJQ0MgUHJvZmlsZ
```

Who exported it like this?

Reply:
Yes, I think I didn't export the image correctly from the Figma design file. I'm looking into correcting this